WS2012R2/lisa/remote-scripts/ica/perf_ntttcp_client.sh

```diff
@@ -612,9 +736,7 @@ else
     echo "ERROR: Unknown test type ${TEST_TYPE}."
     LogMsg "ERROR: Unknown test type ${TEST_TYPE}."
 fi
-zip -r $log_folder.zip $log_folder/*
-
-if [ $sts -eq 0 ]; then
+if ! zip -r "$log_folder.zip" "$log_folder/*"; then
     LogMsg "Test completed"
     echo "Test completed" >> ~/summary.log
 else
```

Comment:
it seems to me that you have duplicated code, this patch needs refactorization. never duplicate code, use functions if you have common code